### PR TITLE
Add lab narrative to eCR Viewer

### LINKED ISSUE #1638: Add Narrative field to Lab Results in eCR Viewer
## Action Requested
Add "Narrative" field to eCR Viewer. This would be associated to each lab result "card". 

## Acceptance Criteria
(not sure)

## Testing
(not sure)

## Background & Context
I accidentally merged Lab Comments and Narrative into just Lab Comments, which isn't right. Narrative is it's own field and can contain a lot of useful info for Lab Results.

## Designs
[Figma Designs](https://www.figma.com/file/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?type=design&node-id=1024%3A136062&mode=design&t=krtjxIZYAAAL6epd-1)
[Figma Clickable Prototype](https://www.figma.com/proto/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?page-id=871%3A2283&type=design&node-id=1024-136062&viewport=725%2C1080%2C0.14&t=QlPigyVJ6G9jvRK3-1&scaling=min-zoom&starting-point-node-id=1606%3A27619&mode=design)

### PR DESCRIPTION
**Pull request by angelathe**
# PULL REQUEST

## Summary
- Add the narrative field to the labs section

- Add seed data that has labs with narrative comments
- Add snapshot tests for labs (note: see my comment)
- modify FHIR conversion to include previously omitted information

## Related Issue
Fixes #1638 

## Screenshots

Note: A ticket was already written removing rows with only `No Data` in the table

https://github.com/CDCgov/phdi/assets/40042932/abfe1293-94a5-43d8-8cde-dcac3b220702

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/src/app/view-data/components/tests/LabInfo.test.tsx [hunk 1]
```diff
@@ -48,6 +54,15 @@ describe("LabInfo", () => {
       ]}
     />
   );
+  it("should match snapshot test", async () => {
+    const labData = evaluateLabInfoData(
```
Comment (angelathe) [inline containers/ecr-viewer/src/app/view-data/components/tests/LabInfo.test.tsx:58]: I don't think using `BundleLabs` is ideal here, but I've already written a separate [issue](https://github.com/CDCgov/phdi/issues/1548) to refactor all the lab-related test fixtures. (I noted this instance in the issue)

**Review by BobanL — COMMENTED** [review]
A couple of nits around snapshot and variable names!

File: containers/fhir-converter/Templates/eCR/Resource/_Observation.liquid [hunk 2]
```diff
@@ -156,6 +157,11 @@
                             "url":"observation entry reference value",
                             "valueString":"{{ observationReferenceValue }}",
                         },
+                    {% elsif observationEntry.value.reference.value -%}
```
[the remark this answers is not in the document]
Comment (angelathe) [reply containers/fhir-converter/Templates/eCR/Resource/_Observation.liquid]: Fixed!

File: containers/fhir-converter/Templates/eCR/Resource/_Observation.liquid [hunk 3]
```diff
@@ -127,6 +127,7 @@
             {% endfor %}
         {% endif %}
         {% assign observationReferenceValue = observationEntry.text.reference.value -%}
+        {% assign observationReferenceValue2 = observationEntry.value.reference.value -%}
```
[the remark this answers is not in the document]
Comment (angelathe) [reply containers/fhir-converter/Templates/eCR/Resource/_Observation.liquid]: Oh gosh sorry I forgot to finish editing the changes -- thanks for catching!! Fixed.

File: containers/ecr-viewer/src/app/view-data/components/tests/__snapshots__/LabInfo.test.tsx.snap
[the remark this answers is not in the document]
Comment (angelathe) [reply containers/ecr-viewer/src/app/view-data/components/tests/__snapshots__/LabInfo.test.tsx.snap:1]: Fixed

**Review by BobanL — APPROVED** [review]
🎉

**Pull request merged**